include/python/spartaModules.hpp

```diff
@@ -161,6 +161,10 @@ namespace sparta
                     return self.getChildAs<ParameterBase>(name, true);
                 },
                 py::return_value_policy::reference);
+        
```

Comment:
The better way is "def_property_readonly" to get value directly rather than .get() API